localization/bg/strings.json

```
@@ -553,6 +559,7 @@
         },
         "add_custom_link": "Добавяне на персонализирана връзка",
         "auto_detect": "Автоматично засичане"
```

You need to look that you have a trailing comma here, on all lines except for the last one in an object ;)
```suggestion
         "auto_detect": "Автоматично засичане",
```